reference/git_github.rst

```diff
@@ -3,6 +3,16 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
+Best practises
+--------------
+- Create a separate pull request for each bug. If you have multiple commits in one pull request, squash them into one.
```

Comment:
While true, beginners do not work on complex changes and except for the case you described, I think it's fair to say that we want to have one commit only.